torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -292,27 +277,26 @@ bool Reducer::ddp_graph_static() {
 }
 
 void Reducer::initialize_local_used_map() {
-  const auto replica_count = replicas_.size();
-  const auto variable_count = replicas_[0].size();
+  const auto replica_count = 1;
```

Comment:
nit: remove this?

Reply:
I originally left it to keep the appropriate context for lines 282-283, but I can remove it if you think it is cleaner